symphony/integration/conductor-tenancy-sanitizer/server/src/main/main10.iml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think this file was created by IntelliJ and shouldn't be committed?

Reply:
Thanks for catching this, fixed.